commit b2324267a6e96d6513d4ec7799c251b5aae5f4c0
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Thu Oct 7 19:52:06 2010 -0700

    staging: brcm80211: Don't include bitfuncs.h
    
    Remove include of bitfuncs.h, as it's completely uncessary for these files.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/phy/wlc_phy_n.c
+++ b/drivers/staging/brcm80211/phy/wlc_phy_n.c
@@ -22,7 +22,6 @@
 #include <osl.h>
 #include <siutils.h>
 #include <sbchipc.h>
-#include <bitfuncs.h>
 #include <hndpmu.h>
 #include <bcmendian.h>
 
